web/src/rendered_markdown.js

```diff
@@ -137,6 +139,22 @@ export const update_elements = ($content) => {
         }
     });
 
+    $content.find(".topic-mention").each(function () {
+        const $message_row = $content.closest(".message_row");
+        const message_id = rows.id($message_row);
+        const message = message_store.get(message_id);
+        const stream_id = message.stream_id;
+
+        // Highlight the mention only if the user is subscribed to the stream and
+        // is a topic participant.
+        if (
+            stream_data.is_user_subscribed(stream_id, people.my_current_user_id()) &&
+            message.topic_wildcard_mentioned
```

Comment:
I think this logic is wrong -- can't we just check `message.topic_wildcard_mentioned`? That answers the question of whether the mention applied to you or not, without any reference to subscriptions.

Reply:
If a user unsubscribes. Should we highlight previous topic mentions in that stream?